arch/arm/core/aarch32/cortex_m/Kconfig

```
@@ -96,6 +96,13 @@ config DCACHE_LINE_SIZE
 config ICACHE_LINE_SIZE
 	default 32
 
+config DEBUG_MONITOR
```

DebugMonitor exception is a Cortex-M-specific feature and should be prefixed accordingly.


```suggestion
config CORTEX_M_DEBUG_MONITOR
```